src/uwtools/tests/drivers/test_fv3.py

```diff
@@ -132,7 +132,7 @@ def test_FV3_field_table(driverobj):
     src.touch()
     dst = driverobj._rundir / "field_table"
     assert not dst.is_file()
-    driverobj._driver_config["field_table"] = {"base_file": src}
+    driverobj._driver_config["field_table"] = {"base_file": str(src)}
```

Comment:
Just updating the test to hopefully mimic real-life behavior a bit more closely.